main.lua

```diff
@@ -393,20 +398,31 @@ function LSPClient:handleResponseResult(method, result)
         else
             display_info("WARNING: Ignored textDocument/hover result due to unrecognized format")
         end
+
+    --- TODO merge textDocument/formatting and textDocument/rangeFormatting?? Same response
```

Comment:
I'd rather keep them separate because it's only a few lines of duplicated code, and they have a different display message.